src/Pdf/Engine/WkHtmlToPdfEngine.php

```diff
@@ -114,7 +114,11 @@ protected function _getCommand()
 
         $command = $this->_binary;
         foreach ($options as $key => $value) {
-            if (empty($value)) {
+            if (is_array($value)) {
```

Comment:
I would have kept the empty check first, as an empty array also doesnt make much sense right?